[swagger-codegen-3.0.0-rc0] Build and swagger-codegen-cli.jar execution issues

##### Description
When I build swagger-codegen-3.0.0-rc0 using "mvn clean package" there are errors and also when I run /swagger-codegen-cli.jar generate

##### Swagger-codegen version
swagger-codegen-3.0.0-rc0 from https://github.com/swagger-api/swagger-codegen/releases/tag/v3.0.0-rc0

##### Swagger declaration file content or url: Cutdown Pet.yaml:
```yaml
openapi: "3.0.0"
info:
  version: 1.0.0
  title: Swagger Petstore
servers:
  - url: http://127.0.0.1:8080
paths:
  /pets:
    get:
      summary: Return the pet
      operationId: getThePet
      responses:
        '200':
          description: A pet
          content:
            application/json:    
              schema:
                $ref: "#/components/schemas/Pet"
components:
  schemas:
    Pet:
      required:
        - id
        - name
      properties:
        id:
          type: integer
          format: int64
        name:
          type: string
      example:
        id: 1
        name: "Trevor"
  ```

##### Command line used for generation
java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -l java -i ../pet.yaml -o C:/tmp/server/petstore3/java

##### Build errors (duplicate output lines removed)
mvn clean package
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] swagger-codegen-project
[INFO] swagger-codegen (core library)
[INFO] swagger-codegen (executable)
[INFO] swagger-codegen (maven-plugin)
[INFO] swagger-generator
[INFO] Building swagger-codegen-project 3.0.0-rc0
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ swagger-codegen-project ---
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce-maven) @ swagger-codegen-project ---
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce-versions) @ swagger-codegen-project ---
[INFO] --- maven-surefire-plugin:2.19.1:test (test-testng) @ swagger-codegen-project ---
[INFO] No tests to run.
[INFO] --- maven-dependency-plugin:2.8:copy-dependencies (default) @ swagger-codegen-project ---

...

[INFO] --- maven-surefire-plugin:2.19.1:test (default-test) @ swagger-codegen ---

-------------------------------------------------------
 T E S T S
Results :
Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO] --- maven-surefire-plugin:2.19.1:test (test-testng) @ swagger-codegen ---
Running TestSuite
[main] INFO org.reflections.Reflections - Reflections took 266 ms to scan 2 urls, producing 43 keys and 232 values
[main] INFO org.reflections.Reflections - Reflections took 159 ms to scan 2 urls, producing 43 keys and 232 values
[main] INFO io.swagger.codegen.ignore.CodegenIgnoreProcessor - No .swagger-codegen-ignore file found.
[main] WARN io.swagger.codegen.DefaultCodegen - generated unique operationId `duplicate_0`
[main] WARN io.swagger.codegen.DefaultCodegen - Empty operationId found for path: get /pet. Renamed to auto-generated operationId: petGet
[main] WARN io.swagger.codegen.ignore.CodegenIgnoreProcessor - Output directory does not exist, or is inaccessible. No file (.swagger-codegen-ignore) will be evaluated.
[main] ERROR io.swagger.codegen.languages.AbstractJavaCodegen - No Type defined for Property Pet
[main] ERROR io.swagger.codegen.DefaultCodegen - String to be sanitized is null. Default to Object
[main] ERROR io.swagger.codegen.languages.AbstractJavaCodegen - No Type defined for Property Order
[main] ERROR io.swagger.codegen.languages.AbstractJavaCodegen - No Type defined for Property User
the size is: 2
[main] WARN io.swagger.codegen.DefaultCodegen - escapeUnsafeCharacters should be overridden in the code generator with proper logic to escape unsafe characters
[main] WARN io.swagger.codegen.DefaultCodegen - escapeQuotationMark should be overridden in the code generator with proper logic to escape single/double quote
Tests run: 45, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 3.382 sec - in TestSuite

##### Execution errors running swagger-codegen-cli.jar generate
java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -l java -i ../pet.yaml -o C:/tmp/server/petstore3/java
[Thread-0] INFO io.swagger.codegen.ignore.CodegenIgnoreProcessor - No .swagger-codegen-ignore file found.
[Thread-0] ERROR io.swagger.codegen.DefaultCodegen - String to be sanitized is null. Default to Object
Exception in thread "Thread-0" java.lang.RuntimeException: Could not generate model 'Pet'
        at io.swagger.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:407)
        at io.swagger.codegen.DefaultGenerator.generate(DefaultGenerator.java:722)
        at io.swagger.codegen.cmd.Generate.run(Generate.java:312)
        at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.io.FileNotFoundException: /v2/Java/\model.mustache
        at com.github.jknack.handlebars.io.URLTemplateLoader.sourceAt(URLTemplateLoader.java:70)
        at com.github.jknack.handlebars.Handlebars.compile(Handlebars.java:357)
        at com.github.jknack.handlebars.Handlebars.compile(Handlebars.java:343)
        at io.swagger.codegen.DefaultGenerator.getHandlebars(DefaultGenerator.java:1013)
        at io.swagger.codegen.DefaultGenerator.processTemplateToFile(DefaultGenerator.java:738)
        at io.swagger.codegen.DefaultGenerator.generateModels(DefaultGenerator.java:394)
        ... 3 more

--- a/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java
+++ b/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java
@@ -1010,7 +1010,8 @@ public class DefaultGenerator extends AbstractGenerator implements Generator {
         final Handlebars handlebars = new Handlebars(templateLoader);
         config.addHandlebarHelpers(handlebars);
 
-        return handlebars.compile(templateFile.replace(".mustache", StringUtils.EMPTY));
+        templateFile = templateFile.replace(".mustache", StringUtils.EMPTY).replace("\\", "/");
+        return handlebars.compile(templateFile);
     }
 
     private boolean isJavaCodegen(String name) {

